README.md

```diff
@@ -33,28 +33,38 @@ First, be sure to register for an [airtable](https://airtable.com) account, crea
 ```ruby
 # Pass in api key to client
 @client = Airtable::Client.new("keyPCx5W")
+# Or if you have AIRTABLE_KEY varibale you can use
```

Comment:
typo: varibale -> variable

is this referring to an environment variable? if so, let's make that clearer

Reply:
Done